core/vm/opcodes.go

```diff
@@ -200,7 +203,7 @@ const (
 
 // unofficial opcodes used for parsing.
 const (
-	PUSH OpCode = 0xb0 + iota
+	PUSH OpCode = 0xc0 + iota
```

Comment:
Oops?

Reply:
Not an "oops", but I can revert that now since we moved the opcodes